modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/SeedlistEntry.scala

```diff
@@ -0,0 +1,90 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.Order
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.SeedlistEntry.Alias
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import com.comcast.ip4s.IpAddress
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.auto._
+import fs2.data.csv._
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import io.estatico.newtype.macros.newtype
+
+case class SeedlistEntry(
+  peerId: PeerId,
+  ipAddress: Option[IpAddress],
+  alias: Option[Alias],
+  bias: Option[TrustValueRefined],
+  ordinalTimeline: Option[HistoricalSnapshotOrdinalEntries]
+)
+
+object SeedlistEntry {
+
+  @derive(eqv)
+  @newtype
+  case class Alias(value: String)
+
+  object Alias {
+
+    implicit val optionalAliasCellDecoder: CellDecoder[Option[Alias]] =
+      CellDecoder.stringDecoder.map { s =>
+        Option.when(s.trim.nonEmpty)(Alias(s.trim))
+      }
+
+  }
```

Comment:
I don't know how this class is intended to be used so FWIW, the decoder is not allowing leading/trailing whitespace in the value but there's nothing to stop the programmer from doing `Alias("  foo  ")`

Reply:
@sdavidp what's wrong with that? @cngo-github you can use the refined type here if you want.
anyway, our `Hash` case class is also `(value: String)` so the programmer can do `Hash("   abc   ")` although the hash would be invalid 🤷

Reply:
Changed to

```scala
  type AliasRefinement = Trimmed And NonEmpty
  type AliasRefined = String Refined AliasRefinement

  @derive(eqv)
  @newtype
  case class Alias(value: AliasRefined)
```